scripts/install_third_party_libs.py

```diff
@@ -87,12 +87,13 @@
 BUF_DIR = os.path.join(
     common.OPPIA_TOOLS_DIR, 'buf-%s' % common.BUF_VERSION)
 PROTOC_DIR = os.path.join(BUF_DIR, 'protoc')
-# List of a tuple represents the path of files that needs to be compiled by
-# protobuf and a boolean value which represents whether buf needs to generate
-# its js and ts file.
-PROTO_FILES_PATHS = [
-    (True, os.path.join(common.THIRD_PARTY_DIR, 'oppia-ml-proto-0.0.0')),
-    (False, common.THIRD_PARTY_OPPIA_ANDROID_PROTO_DIR)]
+# A list of 2-tuples, each representing a proto file. For each tuple,
```

Comment:
Please update this comment based on the changes  that have been made to the constant ...

Reply:
Updated.